### DISCUSSION TITLE
After upgrading from nuxt 3.7.0 to 3.11.2 the application hangs

### DISCUSSION THREAD

**Discussion by dustin-auby** [nuxt/nuxt#27101] [General]
Hi all,

After upgrading from nuxt 3.7.0 to 3.11.2 the application hangs. i have tried stripping down to as basic as possible where i just have a button on the page and it seems as soon as i use a slot componebt it hangs the page

`
<template>
    <v-dialog v-model="visible" :max-width="maxWidth + 'px'">
        <v-card :class="closeIcon ? 'pt-8' : ''">
            <div v-if="icon != null && icon !== ''"
                class="d-flex flex-row justify-center align-center pa-5 pb-0">
                <v-icon size="40" :color="iconColor">{{ icon }}</v-icon>
            </div>
            <div v-if="closeIcon" class="iconClose">
                <v-btn @click="close" density="comfortable" height="" color="grey" variant="text" icon><v-icon>mdi-close</v-icon></v-btn>
            </div>
            <slot name="default"></slot>
        </v-card>
    </v-dialog>
</template>
`

this <slot name="default"></slot> causes the hang. even if i dont parse anything into it. if i remove the slot it works fine if i add it back the page crashes.

any help would be greatly apriciated as the entire app no longer functions and i would prefer no to have to revert back to the old version

**Comment by TheAlexLichter** [comment]
`<slot name="default"></slot>` is the issue here. `<slot />` should be fine (as that's the default slot)

**Reply by dustin-auby** [reply]
Ok let me try this. should this realy cause a crash though still seems like an issue ?
